examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    # make the nodes aware of each other
</pre><p>maybe it would be nicer to do this during the simulation whenever they are in line of sight? similar to the constellation notebook where I update the known actors for each sat every iteration?</p><p> </p><p>Otherwise it may seem like you ignore line of sight </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>